@xen-orchestra/lite/src/views/new-vm/NewVmView.vue

```diff
@@ -0,0 +1,989 @@
+            </div>
+          </div>
+          <!-- MEMORY SECTION -->
+          <UiTitle>{{ $t('new-vm.memory') }}</UiTitle>
+          <div class="memory-container">
+            <UiInput v-model="vmState.vCPU" accent="brand" href="''">{{ $t('new-vm.vcpu') }}</UiInput>
+            <!-- TODO remove (GB) when we can use new selector -->
+            <UiInput v-model="ramFormatted" accent="brand" href="''">{{ $t('new-vm.ram') }} (GB)</UiInput>
+            <UiInput v-model="vmState.topology" accent="brand" href="''" disabled>{{ $t('new-vm.topology') }}</UiInput>
+          </div>
+          <!-- NETWORK SECTION -->
+          <UiTitle>{{ $t('new-vm.network') }}</UiTitle>
+          <div class="network-container">
+            <VtsTable vertical-border>
+              <thead>
+                <tr>
+                  <th id="interfaces">
+                    <VtsIcon accent="current" :icon="faNetworkWired" />
+                    {{ $t('new-vm.interfaces') }}
+                  </th>
+                  <th id="mac_addresses">
+                    <VtsIcon accent="current" :icon="faAt" />
+                    {{ $t('new-vm.mac-addresses') }}
+                  </th>
+                  <th id="delete" />
+                </tr>
+              </thead>
+              <tbody>
+                <tr v-for="(network, index) in vmState.networkInterfaces" :key="index">
+                  <td>
+                    <FormSelect v-model="network.interface">
+                      <optgroup :label="poolName">
+                        <option v-for="nw in networks" :key="nw.$ref" :value="nw.$ref">
+                          {{ nw.name_label }}
+                        </option>
+                      </optgroup>
+                    </FormSelect>
+                  </td>
+                  <td>
+                    <UiInput v-model="network.macAddress" placeholder="Auto-generated" accent="brand" />
+                  </td>
+                  <td>
+                    <UiButtonIcon
+                      :icon="faTrash"
+                      size="medium"
+                      accent="brand"
+                      variant="secondary"
+                      @click="vmState.networkInterfaces.splice(index, 1)"
+                    />
+                  </td>
+                </tr>
+                <tr>
+                  <td colspan="3">
+                    <UiButton
+                      :left-icon="faPlus"
+                      variant="tertiary"
+                      accent="brand"
+                      size="medium"
+                      @click="addNetworkInterface"
+                    >
+                      {{ $t('new-vm.new') }}
+                    </UiButton>
+                  </td>
+                </tr>
+              </tbody>
+            </VtsTable>
+          </div>
+          <!-- STORAGE SECTION -->
+          <UiTitle>{{ $t('new-vm.storage') }}</UiTitle>
+          <div class="storage-container">
```

Comment:
Not used:
```suggestion
          <div>
```
Check if div is useful